Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM jupyter/base-notebook:latest
+
+LABEL maintainer="Fabian Brandt-Tumescheit <brandtfa@hu-berlin.de>"
+
+USER root
+RUN mkdir /var/lib/apt/lists/partial && \
+    apt-get update && \
+    apt-get install -y  --no-install-recommends cmake git build-essential && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+USER 1000
+RUN pip install --upgrade pip setuptools cython
+RUN pip install powerlaw
+RUN pip install sklearn
+RUN pip install seaborn
+RUN pip install pandas
+RUN pip install tabulate
+RUN pip install matplotlib
+RUN pip install networkx
+RUN pip install ipycytoscape
+
+USER root
+RUN mkdir -p ${HOME}/networkit
+COPY . ${HOME}/networkit/
+RUN mv ${HOME}/networkit/input ${HOME}
+RUN mv ${HOME}/networkit/notebooks ${HOME}
+RUN rm ${HOME}/notebooks/test_notebooks.py
```

Comment:
I still think that this is ugly. If we think that this file is distracting, we should move it to a different directory of the repo.

Reply:
Moved it now instead of deleting. The reasoning for removing the file from the folder is to have focus on the first-time experience (which is likely the target-group for binder-notebooks).